src/LuckyBuy.sol

```diff
@@ -894,6 +1041,34 @@ contract LuckyBuy is
         emit MinRewardUpdated(oldMinReward, minReward_);
     }
 
+    /// @notice Sets the bulk commit fee. Is a percentage fee applied to bulk commits
+    /// @param bulkCommitFee_ New bulk commit fee in basis points
+    /// @dev Only callable by ops role
+    /// @dev Emits a BulkCommitFeeUpdated event
+    function setBulkCommitFee(
+        uint256 bulkCommitFee_
+    ) external onlyRole(OPS_ROLE) {
+        _setBulkCommitFee(bulkCommitFee_);
+    }
+
+    function _setBulkCommitFee(uint256 bulkCommitFee_) internal {
+        if (bulkCommitFee_ > BASE_POINTS) revert InvalidBulkCommitFee();
+        uint256 oldBulkCommitFee = bulkCommitFee;
+        bulkCommitFee = bulkCommitFee_;
+        emit BulkCommitFeeUpdated(oldBulkCommitFee, bulkCommitFee_);
+    }
+
+    /// @notice Sets the maximum bulk size for commit operations.
+    /// @param maxBulkSize_ New maximum bulk size.
+    /// @dev Only callable by admin role.
+    /// @dev Emits a MaxBulkSizeUpdated event.
+    function setMaxBulkSize(
+        uint256 maxBulkSize_
+    ) external onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
Can we pls emit an event setMaxBulkSize? You caught it in the other functions I think.